server/src/services/tooljet_db.service.ts

```diff
@@ -259,14 +264,29 @@ export class TooljetDbService {
 
     if (!internalTable) throw new NotFoundException('Internal table not found: ' + tableName);
 
-    let query = `ALTER TABLE "${internalTable.id}" ADD ${column['column_name']} ${column['data_type']}`;
-    if (column['column_default']) query += ` DEFAULT ${this.addQuotesIfString(column['column_default'])}`;
-    if (column?.constraints_type?.is_primary_key ?? false) query += ` PRIMARY KEY`;
-    if (column?.constraints_type?.is_not_null ?? false) query += ` NOT NULL`;
+    const tjdbQueryRunnner = this.tooljetDbManager.connection.createQueryRunner();
+    await tjdbQueryRunnner.connect();
+    await tjdbQueryRunnner.startTransaction();
 
-    const result = await this.tooljetDbManager.query(query);
-    await this.tooljetDbManager.query("NOTIFY pgrst, 'reload schema'");
-    return result;
+    try {
+      await tjdbQueryRunnner.addColumn(
+        internalTable.id,
+        new TableColumn({
+          name: column['column_name'],
+          type: column['data_type'],
+          ...(column['column_default'] && { default: this.addQuotesIfString(column['column_default']) }),
```

Comment:
I think `this.addQuotesIfString` won't be necessary as TypeORM will handle it and we are not generating the SQL query explicitly